WordPressCom-Stats-iOS/UI/NSObject+StatsBundleHelper.h

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
+
+@interface NSObject (StatsBundleHelper)
+
+@property (nonatomic, readonly) NSBundle *statsBundle;
```

Comment:
Just wondering if this should be in `NSObject` or `UIView` (and whether we wanna prefix this or not!)